resources/views/dashboard/admin/realops/index.blade.php

```diff
@@ -47,8 +47,8 @@
                 <td>{{ $f->dep_time_formatted }}</td>
                 <td>{{ $f->dep_airport }}</td>
                 <td>{{ $f->arr_airport }}</td>
-                @if($f->est_arr_time)
-                    <td>{{ $f->est_arr_time_formatted }}</td>
+                @if($f->est_time_enroute)
+                    <td>TEST TEST {{ $f->est_time_enroute_formatted }}</td>
```

Comment:
Looks like `TEST TEST` got left in here...

Reply:
Oops :sweat_smile: will fix.

Reply:
Removed